routes/api/user.js

```diff
@@ -3,10 +3,38 @@ const router = express.Router();
 const userController = require("../../controller/controllerUser");
 const validation = require("../../middlewares/validationUser");
 const authMiddleware = require("../../middlewares/jwt");
+const multer = require("multer");
+
+const storage = multer.diskStorage({
+	destination: "tmp/",
+	filename: (req, file, cb) => cb(null, file.originalname),
+	limits: { fileSize: 1048576 },
+});
+
+const mimeTypeAllowedList = [
+	"image/png",
+	"image/jpg",
+	"image/jpeg",
+	"image/gif",
+];
+
+const multerInstance = multer({
+	storage,
+	fileFilter: (req, file, cb) => {
+		const mimetype = file.mimetype;
+		if (!mimeTypeAllowedList.includes(mimetype)) {
+			return cb(null, false);
+		}
+		return cb(null, true);
```

Comment:
to jest fajny kandydat na przepisanie :)

mysle, ze: 
```js
const isMimetypeAllowed = mimeTypeAllowedList.includes(mimetype);
return cb(null, isMimetypeAllowed );
```

byloby czystsze i prostsze. Bardziej "human-readable", rowniez :)